go.mod

```diff
@@ -21,45 +14,45 @@ require (
 	github.com/jarcoal/httpmock v1.3.0
 	github.com/jpillora/backoff v1.0.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/ginkgo/v2 v2.19.0
-	github.com/onsi/gomega v1.33.1
+	github.com/onsi/ginkgo/v2 v2.22.2
+	github.com/onsi/gomega v1.36.2
 	github.com/openshift-online/ocm-sdk-go v0.1.422
-	github.com/openshift/api v0.0.0-20240530231226-9d1c2e5ff5a8
+	github.com/openshift/api v0.0.0-20241104230711-4c27e61e5554
 	github.com/openshift/client-go v0.0.0-20240510131258-f646d5f29250
 	github.com/openshift/cluster-version-operator v1.0.1-0.20230322131514-c659e6ccaca7
 	github.com/openshift/library-go v0.0.0-20240517135010-e93e442c2b18
 	github.com/openshift/machine-config-operator v0.0.1-0.20230330142923-2832f049b3f4
```

Comment:
Can you update the recent CVO and MCO as well as per 4.18 release? We will need to test and validate the upgrades accordingly then.